cds_ils/importer/serializers/schema.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would have put loaders in `loaders/jsonschemas/importer.py` as we normally do.
I would also add:
```
    class Meta:
        """Meta attributes for the schema."""

        unknown = EXCLUDE
```
in the main Schema class so that extra keys are excluded.